packages/keymaps/src/browser/keybindings-widget.tsx

```diff
@@ -669,6 +690,16 @@ export class KeybindingWidget extends ReactWidget {
         }
     }
 
+    protected handleContextMenu = (item: KeybindingItem, e: React.MouseEvent | React.KeyboardEvent): void => {
```

Comment:
It does not make sense to capture a method instance as property of it is not passed as a reference to event handler, but a new instance created each time